fix: config store documentation fix (#1160) (#1161)
fastly/js-compute-runtime: documentation/docs/config-store/ConfigStore/ConfigStore.mdx

# `ConfigStore()`

> The **`ConfigStore` constructor** lets you access a specific [Fastly Edge ConfigStore](https://www.fastly.com/documentation/reference/api/services/resources/config-store/).

Reviewer: ```suggestion
The **`ConfigStore` constructor** lets you access a specific [Fastly Config Store](https://www.fastly.com/documentation/reference/api/services/resources/config-store/).
```

### Parameters

> - `name` _: string_

Author: placeholder for now, similar wording around the `name` parameter should be applied to other service-linked resource types

>   - Defines a Config Store instance using the resource link name.

Reviewer 2: The resource stores are weird in that we capitalize when talking about the feature but use lowercase when talking about the object.
```suggestion
  - Defines a config store instance using the resource link name.
```

>   - The name of the [Fastly Edge ConfigStore](https://www.fastly.com/documentation/reference/api/services/resources/config-store/) that this `ConfigStore` instance should provide access to.

Reviewer: Guess what you've stepped into without realizing it... :-)

As we're discussing in another thread right now, this is incorrect, as the name provided to the constructor must be the name of the service's *resource link* which links to the Config Store, not the name of the Config Store itself. This is also true for KV Stores and Secret Stores, so whatever language we come up with to address that here should be applied to those resource types too.
```suggestion
  - The name of the [Fastly Config Store](https://www.fastly.com/documentation/reference/api/services/resources/config-store/) that this `ConfigStore` instance should provide access to.
```

Author: So we can either change the wording and drop a breakcrumb so the wording eventually goes to the source.
 `
The name provided must be the service's resource link [name](https://www.fastly.com/documentation/guides/concepts/edge-state/dynamic-config/#config-stores).
`
or
Add a note 
**Note** The name is the service's resource link name which links to the Config Store, not the name of the Config Store itself.

I prefer the first but I'll include either in the other resources docs as well.

Reviewer: I'll defer to @gaylin for that one.

Reviewer 3: @dbarshop , could you weigh in here please on the terminology that you've worked out with Eng and Legal in the past?

Reviewer 4: (commenting here to subscribe to the thread and see the outcome for the Rust SDK as well)

## Examples

> In this example we have a ConfigStore named "animals" and we return the "cat" entry as the response body to the client.

Reviewer: ```suggestion
In this example we have a Config Store named "animals" and we return the "cat" entry as the response body to the client.
```

> In this example we have a Config Store named "animals" and we return the "cat" entry as the response body to the client.

Reviewer: ```suggestion
In this example we have a resource link named "animals" (which is linked to a Config Store) and we return the "cat" entry as the response body to the client.
```

> In this example we have a resource link named "animals" (which is linked to a Config Store) and we return the "cat" entry as the response body to the client.

Reviewer 2: ```suggestion
In this example we have a resource link named "animals" (which is linked to a config store) and we return the "cat" entry as the response body to the client.
```
